src/packages/@ncigdc/containers/explore/ExplorePage.js

```diff
@@ -153,41 +156,55 @@ export const ExplorePageComponent = (props: TProps) =>
       },
     ]}
     results={
-      <TabbedLinks
-        queryParam="searchTableTab"
-        defaultIndex={0}
-        links={[
-          {
-            id: 'cases',
-            text: `Cases (${props.viewer.explore.cases.hits.total.toLocaleString()})`,
-            component: !!props.viewer.explore.cases.hits.total
-              ? <CasesTab />
-              : <NoResultsMessage>No Cases Found.</NoResultsMessage>,
-          },
-          {
-            id: 'genes',
-            text: `Genes (${props.viewer.explore.genes.hits.total.toLocaleString()})`,
-            component: props.viewer.explore.genes.hits.total
-              ? <GenesTab viewer={props.viewer} />
-              : <NoResultsMessage>No Genes Found.</NoResultsMessage>,
-          },
-          {
-            id: 'mutations',
-            text: `Mutations (${props.viewer.explore.ssms.hits.total.toLocaleString()})`,
-            component: props.viewer.explore.ssms.hits.total
-              ? <MutationsTab
-                  totalNumCases={props.viewer.explore.cases.hits.total}
-                  viewer={props.viewer}
-                />
-              : <NoResultsMessage>No Mutations Found.</NoResultsMessage>,
-          },
-          {
-            id: 'oncogrid',
-            text: 'OncoGrid',
-            component: <OncogridTab />,
-          },
-        ]}
-      />
+      <span>
+        <Row
```

Comment:
This row isn't doing anything besides the padding right? Could remove it and apply the padding to the button.